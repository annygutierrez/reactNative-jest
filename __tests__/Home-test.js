import 'react-native';
import React from 'react';
import Home from '../App.js';
import renderer from 'react-test-renderer';

test('Home snapShot', () => {
    const snap = renderer.create(
        <Home />
    ).toJSON();

    expect(snap).toMatchSnapshot();
})