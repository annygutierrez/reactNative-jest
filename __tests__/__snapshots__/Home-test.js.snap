// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapShot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#F5FCFF",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 24,
          "margin": 10,
          "textAlign": "center",
        }
      }
    >
      Profile Component
    </Text>
  </View>
</View>
`;
